lib/membrane_webm/parser/webm.ex

```diff
@@ -121,30 +145,35 @@ defmodule Membrane.WebM.Parser.WebM do
     # track_number is a vint with size 1 or 2 bytes
     {:ok, {track_number, body}} = EBML.decode_vint(bytes)
 
-    <<timecode::integer-signed-big-size(16), keyframe::1, reserved::3, invisible::1, lacing::2,
+    <<timecode::integer-signed-big-size(16), keyframe::1, _reserved::3, invisible::1, lacing::2,
       discardable::1, data::binary>> = body
 
-    lacing =
-      case lacing do
-        0b00 -> :no_lacing
-        0b01 -> :Xiph_lacing
-        0b11 -> :EBML_lacing
-        0b10 -> :fixed_size_lacing
-      end
-
-    # TODO: deal with lacing != 00 https://tools.ietf.org/id/draft-lhomme-cellar-matroska-04.html#laced-data-1
-
     %{
       track_number: track_number,
       timecode: timecode,
       header_flags: %{
         keyframe: keyframe == 1,
-        reserved: reserved,
         invisible: invisible == 1,
-        lacing: lacing,
+        lacing: check_lacing_mode(lacing),
         discardable: discardable == 1
       },
       data: data
     }
   end
+
+  # TODO: handle laced data
```

Comment:
If we will not add support for lacing in this PR, we can remove this TODO and add an issue in the repo.